src/components/SelectInput/SelectInput.stories.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
```

Comment:
@JCarmine when you get a chance check out this PR for a better way to use state in stories. This way the source code is preserved with the true components for users to peruse.

https://github.com/palmetto/palmetto-components/pull/63